src/utilities/cloudinaryUpload.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+
+
+const uploadImageCloudinary = async (value) => {
+    try {
+        const formData = new FormData();
+        formData.append('upload_preset', 'zeooiv7x');
+        formData.append('file', value);
+        const response = await axios.post('https://api.cloudinary.com/v1_1/ah-med/image/upload', formData);
+        return response.data.secure_url;
+    }
```

Comment:
Good job on creating this utility @nwashangai, I feel you should have used the utility created in the "apiRequest.js" file to make the "axios.post" request on line 9. the utility file is used for making axios requests.